### ISSUE TITLE
Unable to setup ACMS using the composer template

### ISSUE THREAD

**Issue by joshirohit100** [acquia/acquia_cms#807]
I am trying to setup the ACMS using this composer template https://github.com/acquia/acquia-cms-project but I am getting error.

### Command I used for creating project -
composer create-project acquia/acquia-cms-project

### Error I am getting -
```
Applying patches for drupal/acquia_telemetry-acquia_telemetry
    https://www.drupal.org/files/issues/2020-08-18/3165473-27.patch (Add check for core index)

Applying patches for acquia/cohesion
    https://gist.githubusercontent.com/chandan-singh7929/c70de6c88b8951022786cfeed75ff714/raw/dfe8481b77594405b27af24ec88da8254d9a00c1/frontendBuilderDropzone.patch (Prevent error call to a member function getOption() on null during starter enable)
   Could not apply patch! Skipping. The error was: Cannot apply patch https://gist.githubusercontent.com/chandan-singh7929/c70de6c88b8951022786cfeed75ff714/raw/dfe8481b77594405b27af24ec88da8254d9a00c1/frontendBuilderDropzone.patch

                                                                                                                                                                                
  [Exception]                                                                                                                                                                   
  Cannot apply patch Prevent error call to a member function getOption() on null during starter enable (https://gist.githubusercontent.com/chandan-singh7929/c70de6c88b8951022  
  786cfeed75ff714/raw/dfe8481b77594405b27af24ec88da8254d9a00c1/frontendBuilderDropzone.patch)!                                                                                  
                                                                                                                                                                                

create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--ask] [--] [<package>] [<directory>] [<version>]
```

**Comment by nagwani** [comment]
@joshirohit100 Please try to setup now  again. We have just released 1.2.2 and that should fix the issue.

**Comment by joshirohit100** [comment]
Thanks! Now working fine for me.
